AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/test_values/phsyx_collider_test_values.py

```diff
@@ -0,0 +1,58 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+These test values are intended to be utilized alongside the PhysX Collider
+"""
+
+CYLINDER_SUBDIVISION_TESTS = {
+    # "Negative Value Test": (-1, False), # o3de/o3de#12608 - Crash if subdivision set below 3
+    # "Zero Value Test": (0, False), # o3de/o3de#12608 - Crash if subdivision set below 3
+    # "Out Of Lower Bounds Test": (2, False), # o3de/o3de#12608 - Crash if subdivision set below 3
+    "Minimum Value Test": (3, True),
+    "Middle(ish) Value Test": (60, True),
```

Comment:
nit: Should we use 'ish' here? I'd avoid being cute and just give this a more accurate name